session-name-1/slides/Makefile

```diff
@@ -0,0 +1,20 @@
+RVMD = reveal-md
+
+.PHONY: all html pdf clean
+
+all: html pdf
+
+html: _site
+
+_site: slides.md
+	$(RVMD) $< --static $@
```

Comment:
```suggestion
	$(RVMD) --static $@
```

Reply:
How does removing `$<` help? `$<` is a placeholder for `slides.md` (i.e. the rule dependency).